app.libretile.Libretile.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: app.libretile.Libretile
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: libretile.sh
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+
+build-options:
+    build-args:
+      -    --share=network
```

Comment:
This is not allowed, it will not work.

Reply:
ok i will try to make pillow get installed like tkinter does   this might make it so i can fix that ?